verbit/streaming_client.py

```diff
@@ -193,54 +216,122 @@ def _connect_websocket(self, media_config: MediaConfig, response_types: Response
         # build websocket url
         ws_url = self.ws_url + self._get_ws_connect_query_string(media_config=media_config, response_types=response_types)
 
-        try:
+        # create WebSocket instance
+        self._ws_client = WebSocket(enable_multithread=True)
+        # self._ws_client.timeout = 0.01
+        # self._ws_client.timeout = 1.0
```

Comment:
^ temporary, using to test disconnection when WiFi is out etc

Reply:
I'll verify this scenario now, and remove

Reply:
I saw you set it to `20` eventually. Is this the value we actually want?

Reply:
wanted to discuss the `20`, I think it's probably better to not-set-it

Reply:
ok, we decided to expose this as a property and let the users set it according to their needs.